src/oedisi/tools/cli_tools.py

```diff
@@ -213,7 +211,7 @@ def clone_broker(yaml_file_path, target_directory):
     broker_folder = os.path.join(target_directory, "broker")
     if not os.path.exists(broker_folder):
         os.makedirs(broker_folder)
-    copy_tree("./broker", broker_folder)
+    shutil.copytree("./broker", broker_folder, dirs_exist_ok=True)
```

Comment:
dirs_exist_ok=True does not work with Python 3.7. should depreciate support for it

Reply:
pyproject.toml is fixed.